### PR TITLE
[ICDS Dashboard]Data PULL Request

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### PRODUCT DESCRIPTION
https://app.asana.com/0/1160310200927866/1160743483000969

### CODE REVIEW

**Review by mkangia — APPROVED** [review]
i think you can remove the national_level query.

**Comment by mkangia** [comment]
@calellowitz we ran this data pull today and ran
https://github.com/dimagi/commcare-hq/pull/26596/files#diff-f1d4774ece1bc91840f7f70a0bfeb5d8
Just wanted to get your review on this one that it was the correct way to do this.

File: custom/icds_reports/utils/state_level.sql [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+     SUM(num_launched_supervisors) as launched_supervisor,
+     SUM(num_launched_awcs) as launched_awcs
+ FROM awc_location_local left join "agg_awc_daily_2020-02-01" agg_awc_daily_view on awc_location_local.state_id = agg_awc_daily_view.state_id
+ AND awc_location_local.district_id = agg_awc_daily_view.district_id
+  AND awc_location_local.block_id = agg_awc_daily_view.block_id
+   AND awc_location_local.supervisor_id = agg_awc_daily_view.supervisor_id
+    AND awc_location_local.doc_id = agg_awc_daily_view.awc_id
+    AND awc_location_local.aggregation_level = agg_awc_daily_view.aggregation_level
+ WHERE date='2020-02-01' AND awc_location_local.aggregation_level=1 and awc_location_local.state_is_test<>1
+ GROUP BY awc_location_local.state_name;
```
Comment (mkangia) [inline custom/icds_reports/utils/state_level.sql]: @Rohit25negi can you share the reason behind why this change was necessary?
Shubham mentioned that this is needed to protect from a bug.

File: custom/icds_reports/utils/state_level.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
+SELECT
```
Comment (calellowitz) [inline custom/icds_reports/utils/state_level.sql]: It is obviously too late to change this now but why wasn't this already accessible in our existing agg awc daily view that is used elsewhere in the dashboard? https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/migrations/sql_templates/database_views/agg_awc_daily.sql
@ShubhamBansal1997
